src/utils/polygon.h

```diff
@@ -454,6 +457,13 @@ class Polygons
         return paths.size();
     }
 
+    /*!
+     * Convenience function to check if the polygon has no points.
+     *
+     * \return `true` if the polygon has no points, or `false` if it does.
```

Comment:
Correct description.

I tend to just not document functions like this. It's just too simple for me :P

(Just a comment, no fix requested)